examples/own_dataset/README.md

```diff
@@ -4,10 +4,15 @@
 python train.py dataset.csv --label value1 value2
 ```
 
-## How to use your own dataset
+The `--label` option specifies which row in `dataset.csv` is trained.
```

Comment:
I think it is columns, not rows that we have to specify.

Reply:
Also, as we can specify multiple columns, "is" should be substituted with "are".